src/popup/utils/constants.js

```diff
@@ -65,19 +75,9 @@ export const defaultNetworks = [
     tipContractV2: 'ct_2ZEoCKcqXkbz2uahRrsWeaPooZs9SdCv6pmC4kc55rD4MhqYSu',
     name: 'Testnet',
   },
-  {
-    url: 'https://mainnet.aeternity.io',
-    networkId: 'ae_mainnet',
-    middlewareUrl: 'https://mainnet.aeternity.io/mdw',
-    explorerUrl: 'https://explorer.aeternity.io',
-    compilerUrl: 'https://compiler.aepps.com',
-    backendUrl: 'https://raendom-backend.z52da5wt.xyz',
-    tipContractV1: 'ct_2AfnEfCSZCTEkxL5Yoi4Yfq6fF7YapHRaFKDJK3THMXMBspp5z',
-    name: 'Mainnet',
-  },
 ];
 
-export const defaultNetwork = process.env.NETWORK === 'Testnet' ? defaultNetworks[0] : defaultNetworks[1];
+export const defaultNetwork = process.env.NETWORK === 'Testnet' ? defaultNetworks[1] : defaultNetworks[0];
```

Comment:
Also I suggest to do minor refactoring: instead of having one array `defaultNetworks` I suggest to create 2 separate exportable consts for each of the networks. So here instead of picking them `[0]` or `[1]` you'll be able to use human readable names. Same for the test file `transaction-details.js` - you will be able to import one specific network instead of picking them from array.

So after all:
```
export const NETWORK_MAINNET = { ... }

export const NETWORK_TESTNET = { ... }

export const DEFAULT_NETWORKS = [
  NETWORK_MAINNET,
  NETWORK_TESTNET,
]
```